README.md

```diff
@@ -42,7 +42,7 @@ Other Stuff
 
 For other stuff, including plugins and discussion, check out the [forums](http://forum.mc-server.org) and [Plugin API](http://api-docs.cuberite.org).
 
-Support Us on Gratipay: [![gratipay shield](http://img.shields.io/gratipay/cuberite.svg)](https://www.gratipay.com/cuberite)
+[Support Us on Bountysource (salt)](https://bountysource.com/teams/cuberite)bountysource.com/teams/cuberite
```

Comment:
BS offers badges, but they're not relevant to the recurring-funding part that we're using, they're more for individual issues with bounties on them.